management/server/http/api/openapi.yml

```diff
@@ -562,7 +562,7 @@ components:
           type: integer
           example: 2
         issued:
-          description: How group was issued by API or from JWT token
```

Comment:
This could be:
```
        issued:
          description: How group was issued by API or from JWT token
          type: string
          enum: ["api", "integration", "jwt"]
          example: api
```

Although not sure if `docs` will be able to show it.

Reply:
ok